scheduler.py

```diff
@@ -0,0 +1,87 @@
+"""Module responsible to handle schedules."""
+import time
+from pytz import utc
+
+from uuid import uuid4
+from apscheduler.schedulers.background import BackgroundScheduler
+
+from apscheduler.triggers.cron import CronTrigger
+from kytos.core.helpers import now
+from kytos.core import log
+
+
+class CircuitSchedule:
+    """Schedule events."""
+
+    def __init__(self, **kwargs):
+        """CircuitSchedule contructor."""
```

Comment:
Which object types are expected on kwargs? Maybe an example on the docstring would be enough to clarify that.